### ISSUE TITLE
Publiclab bot.

### ISSUE THREAD

**Issue by Harshithpabbati** [publiclab/plots2#5237]
Bot with features of claiming an issue from comments adding label's to the issue from comments (@publiclabbot claim , @publiclabbot add "bug") in this way.  So that even the new contributers can even label the issues and claim the issue to them. We can even recieve the travis build updates it will label the issue as travis- updates.
Link to the repo is
https://github.com/harshithpabbati/publiclabbot
But I am not as good at webhook's and that's the reason I can't implement it can someone help me in solving this.
I tried to get it from the zulipbot link to it is https://github.com/zulip/zulipbot
Can someone help me out !!! Thanks.

**Comment by Harshithpabbati** [comment]
@publiclab/reviewers @jywarren @gauravano

**Comment by CleverFool77** [comment]
HI @Harshithpabbati I guess There is a probot made by github for these features. We can use it for issue claiming, assigning issues, giving a timeline of 2 weeks to a assigned person and if that contributor didnt made WIP then the contributor would be unassigned.
:thinking: :thinking: :thinking:

**Comment by Harshithpabbati** [comment]
isn't it ????

**Comment by Harshithpabbati** [comment]
Send me the link of it.

**Comment by Harshithpabbati** [comment]
Fine I got it thanks @CleverFool77

**Comment by gauravano** [comment]
https://github.com/probot/probot

**Comment by Harshithpabbati** [comment]
> https://github.com/probot/probot

no I mean the bot similar to this. when we say @publiclabbot claim it should claim the issue and add and remove label.

**Comment by CleverFool77** [comment]
> > https://github.com/probot/probot
> 
> no I mean the bot similar to this. when we say @PublicLabBot claim it should claim the issue and add and remove label.

Yes its possible in probot.. We can do @nameofbot claim. It can help in assigning issues to people. Probot's work is like zulipbot.. 🤔

**Comment by ananya** [comment]
https://github.com/rsarky/pro-claim here it is. 
There are a lot of bots here. We can combine the features of some into one and have our own bot.

**Comment by gauravano** [comment]
If there's a single bot already there for all of our needs then it's great, we can just use that otherwise we can think of creating a bot of our own. What do you all think?

**Comment by Harshithpabbati** [comment]
Yeah mostly things we need is labelling and assigning using comments

**Comment by SamueleEsposito** [comment]
So, will we do this? If we do, I'd love to work on it!
